src/radha/PractisePrograms/PrimeNumber.java

```diff
@@ -0,0 +1,101 @@
+package radha.PractisePrograms;
+
+public class PrimeNumber {
+
+	void validateNoIsPrime(int targetNo) {
+		int num = 2;
+		for (; num <= targetNo/2; num++) {
+			if (targetNo % num == 0) {
+				break;
+			}
+		}
+		if ((targetNo/2)+1 == num)
+			System.out.println(targetNo + " is prime.");
+		else
+			System.out.println(targetNo + " is not prime.");
+	}
+	
+	void booleanNoIsPrime(int targetNo) {
+		boolean flag = true;
+		for (int num = 2; num <= targetNo/2; num++) {
+			if (targetNo % num == 0) {
+				flag = false;
+				break;
+			}
+		}
+		if (flag ==true)
+			System.out.println(targetNo + " is prime.");
+		/*else
+			System.out.println(targetNo + " is not prime.");*/
+	}
+	
+	void validateAllPrimeNo(int target) {
+		for(int number=2; number < target; number++) {
+			booleanNoIsPrime(number);
+		}
+	}
+	
+	void primeRange(int range) {
+		for (int index = 2; index <= range; index++) {
+			int targetNo = index;
+			boolean flag = true;
+			for (int num = 2; num <= targetNo / 2; num++) {
+				if (targetNo % num == 0) {
+					flag = false;
+					break;
+				}
+			}
+			if (flag == true)
+				System.out.println(targetNo + " is prime.");
+			/*
+			 * else System.out.println(targetNo + " is not prime.");
+			 */
+		}
+	}
+	
+	void sumPrimeRange(int startRange, int endRange) {
+		int count = 0, sum =0;
+		for (int targetNo = startRange; targetNo <= endRange; targetNo++) {
+			boolean flag = true;
+			for (int num = 2; num <= targetNo / 2; num++) {
```

Comment:
You don't require to write logic here, you can call that method which you implemented in same class